topology/sof/pipe-low-latency-capture.m4

```diff
@@ -16,12 +16,12 @@ include(`mixercontrol.m4')
 #
 # Volume Mixer control with max value of 32
 C_CONTROLMIXER(PCM PCM_ID Capture Volume, PIPELINE_ID,
-	CONTROLMIXER_OPS(volsw, 256 binds the mixer control to volume get/put handlers, 256, 256),
+	CONTROLMIXER_OPS(volsw, 1 binds the mixer control to volume get/put handlers, 1, 1),
```

Comment:
@RanderWang This seems incorrect. the sof volume_put/get handlers can handle any number of channels up to MAX chan allowed in topology?

Reply:
yes, volume_put/get works with MAX chan. But user only see two channels in user mode. Please
refer to my thesofproject/linux#39

Reply:
this also makes no sense to me. this would require all topology files to be changes to remove this special value of 256 used to bind volume with kcontrols.

Reply:
@RanderWang if the issue is because the default infor function used in topology (volsw) supports only 2 channels, then the right fix for this issue would be have a bespoke info function in SOF as well? as Pierre pointed out, we cannot change the binding for all the topologies that are known to work today.

Reply:
@ranj063 yes, i totally agree with you. But I have pointed out that: if the id is set to 256, the info function cant be binded to a bespoke sof info function. This is the limitation of Asoc

Reply:
@randerwang, i am not sure this is the right thing to do. But can we try "volsw_xr_sw" for info instead of "volsw" to get around the stereo restriction?

Reply:
@ranj063 "volsw_xr_sw" doesn't work and it makes topology loaded failed.
To get info function to be bound with vendor id like 256, a great change will be made
in Asoc. I discussed with Mengdong who is a expert in topology, and got this conclusion.